File: deck/base/code/native/swift/runtime/log.swift
import Foundation

enum log {
    static func writeInfo(_ message: String) { print(message) }
    static func writeWarn(_ message: String) { print(message) }
    static func writeError(_ message: String) { FileHandle.standardError.write((message + "\n").data(using: .utf8)!) }
    static func writeDebug(_ message: String) { print(message) }
}

File: deck/base/code/native/rust/runtime/log.rs
mod log {
    pub fn write_info(message: String) { println!("{}", message); }
    pub fn write_warn(message: String) { println!("{}", message); }
    pub fn write_error(message: String) { eprintln!("{}", message); }
    pub fn write_debug(message: String) { println!("{}", message); }
}
